Framework/DataHandling/src/LoadEventNexus.cpp

```diff
@@ -1157,6 +1183,13 @@ void LoadEventNexus::loadEvents(API::Progress *const prog, const bool monitors)
 
   // if there is time_of_flight load it
   adjustTimeOfFlightISISLegacy(*m_file, m_ws, m_top_entry_name, classType, descriptor.get());
+
+  if (is_time_filtered) {
+    // Now filter out the run and events, using the DateAndTime type.
+    // This will sort both by pulse time
+    m_ws->mutableRun().filterByTime(filter_time_start, filter_time_stop);
```

Comment:
Does `FilterByTime` not do this itself?

Reply:
Yes it does. I'd missed this. So this line can be removed I think

Reply:
Thanks. The unit test already checks that the logs have been filtered and that seems okay so I think we're good.